lib/linguist/languages.yml

```diff
@@ -8388,6 +8388,14 @@ sed:
   ace_mode: text
   tm_scope: source.sed
   language_id: 847830017
+templ:
```

Comment:
Oh yes, and is this really the public human readable representation of the language name… ie all lowercase?

Reply:
https://templ.guide/